cmd/corald/routes/routes.go

```diff
@@ -112,6 +112,12 @@ func routes(a *app.App) {
 				return "/v1/exec/" + c.Params["query_set"]
 			}))
 
+	// Execute xenia queries directly.
+	a.Handle("GET", "/v1/exec/:query_set", handlers.Proxy(xeniadURL, nil))
```

Comment:
I do not understand what you mean with this comment.  

@impronunciable , you need to be able to run already saved xenia queries, not custom ones, right?